### PR TITLE
Minor XrpRpc changes to follow interface more closely

### PR DESCRIPTION
**Pull request by jackbonaguro**

### CODE REVIEW

File: lib/xrp/XrpRpc.js [hunk 1]
```diff
@@ -87,9 +88,9 @@ class XrpRpc {
...
   }
 
-  async sendRawTransactionMany({ signedTxArray }) {
+  async sendRawTransactionMany({ rawTxArray }) {
     let resultArray = [];
-    for (const signedTx of signedTxArray) {
+    for (const signedTx of rawTxArray) {
       const emitData = { signedTx };
       try {
         let txHash = await this.sendRawTransaction({ signedTx });
```
Comment (SethMcGuire) [inline lib/xrp/XrpRpc.js]: This should pass in `rawTx` now

Comment (SethMcGuire) [reply lib/xrp/XrpRpc.js]: Same with line 58

Comment (jackbonaguro) [reply lib/xrp/XrpRpc.js]: Fixed

File: lib/xrp/XrpRpc.js [hunk 2]
```diff
@@ -136,19 +137,22 @@ class XrpRpc {
 
 
   async getRawTransaction({ txid }) {
-    return (await this.asyncCall('tx', [{transaction:txid, binary:true}])).tx;
+    return (await this.asyncRequest('tx', [{transaction:txid, binary:true}])).tx;
```
Comment (micahriggan) [inline lib/xrp/XrpRpc.js]: Random nitpick, I'd prefer 
```
let {tx} = await this.asyncRequest();
return tx;
```

Comment (jackbonaguro) [reply lib/xrp/XrpRpc.js]: @micahriggan I agree; fixed.

**Pull request merged**